src/main.rs

```diff
@@ -121,36 +114,32 @@ fn setup_logger(is_cli: bool, is_debug: bool) -> Result<()> {
         logger = logger.level_for("ajour_core", log::LevelFilter::Trace);
     }
 
-    if is_cli || is_debug {
+    if !is_cli && is_debug {
         logger = logger.chain(std::io::stdout());
     }
 
-    if !is_cli && !is_debug {
-        use std::fs::OpenOptions;
-
-        let config_dir = ajour_core::fs::config_dir();
+    let config_dir = ajour_core::fs::config_dir();
 
-        let log_file = OpenOptions::new()
-            .write(true)
-            .create(true)
-            .append(false)
-            .truncate(true)
-            .open(config_dir.join("ajour.log"))?;
+    let log_file = std::fs::OpenOptions::new()
+        .write(true)
+        .create(true)
+        .append(false)
+        .truncate(true)
+        .open(config_dir.join("ajour.log"))?;
 
-        logger = logger.chain(log_file);
-    };
+    logger = logger.chain(log_file);
```

Comment:
This needs to be reverted. It was working as intended, where we only log to file if not a CLI command and run in release mode. And to stdout if in debug mode or CLI command.

We can change the format for logging when `is_cli` if we want it to look similar to printing directly to std_out, but this gives us way more flexibility in the future if we want to add things like a `verbosity` flag for CLI output by using the stdout chain.